Makefile

```diff
@@ -107,7 +107,10 @@ ${CACHEDIR}/.libraries_token.txt: cache
 	mv $${tmpf} ${CACHEDIR}/.libraries_token.txt
 
 .PHONY: lib-token
-lib-token: ${CACHEDIR}/.libraries_token.txt
+lib-token:
+	if command -v chainctl; then \
+		$(MAKE) ${CACHEDIR}/.libraries_token.txt; \
```

Comment:
Is there a way to glean from a `melange` YAML file that library access is required? If so, maybe the better solution here would be to evaluate the YAML file and only generate a token in that case (and fail hard if a token is needed and `chainctl` is unavailable)

Reply:
That sounds good to me - both for usability and security reasons.